### DISCUSSION TITLE
Github client star repo list still use old username after it changed.

### DISCUSSION THREAD

**Discussion by iseki0** [community/community#35505] [Mobile]
I changed my username long time ago. But the star repo list page still show my old username. And the list is empty. To solve the problem I have to wipe app data.

I can't reproduce it because I don't want to change my username again.

Version: Android 1.83.1
Android: 10

**Comment by jfullstackdev** [comment]
not very sure if it is a cached data, try testing it in Incognito mode

**Comment by iseki0** [comment]
Clear application cache in system app manager is unused. Looks it's not 
a cached in the cache directory

在 2022/10/08 19:16, jdevstatic 写道:

**Comment by iseki0** [comment]
Clear application cache in system app manager is unused. Looks it's not a cached in the cache directory.
